components/FilterBar/selectors.js

```diff
@@ -3,33 +3,42 @@ import {
   getCategoryProductCount,
   getCurrentCategoryChildCount,
 } from '@shopgate/pwa-common-commerce/category/selectors';
+import { getProductsResult } from '@shopgate/pwa-common-commerce/product/selectors/product';
 import { hasActiveFilters } from '@shopgate/pwa-common-commerce/filter/selectors';
 
 /**
- * Selects the summed up shipping costs of the cart.
- * @type {number}
+ * Checks if the filter bar has active filters
+ * @return {bool}
  */
 export const isFilterBarActive = createSelector(
   hasActiveFilters,
   activeFilters => activeFilters
 );
 
 /**
- * Selects the summed up shipping costs of the cart.
- * @type {number}
+ * Checks if the filter bar is shown within a category page
+ * @return {bool}
  */
-export const isFilterBarShown = createSelector(
+export const isFilterBarShownInCategory = createSelector(
```

Comment:
If the selector is specific like that then it should belong to the Category Page, not the FilterBar anymore. And THEN you should rename it back to `isFilterBarShown`.

Reply:
moved to a custom selector